baseapp/msg_service_router.go

```diff
@@ -92,3 +88,10 @@ func (msr *MsgServiceRouter) RegisterService(sd *grpc.ServiceDesc, handler inter
 func (msr *MsgServiceRouter) SetInterfaceRegistry(interfaceRegistry codectypes.InterfaceRegistry) {
 	msr.interfaceRegistry = interfaceRegistry
 }
+
+// gRPC NOOP interceptor
+func noopInterceptor(_ context.Context, _ interface{}, _ *grpc.UnaryServerInfo, _ grpc.UnaryHandler) (interface{}, error) {
```

Comment:
Are you planning to use this function elsewhere? What's the benefit of extracting a tiny snippet of code that just return `nil, nil` when it's not yet reused anywhere else?

This smells like early refactoring, and as such, pointless (best case scenario) if not potentially harmful.

Reply:
Please check the PR description.

> Are you planning to use this function elsewhere?

The function is private

> What's the benefit of extracting a tiny snippet  of code

improving readability

Reply:
Does it really improve readability to extract an _empty function_? I too tend to prefer named functions, though it generally boils to one question to me: _Am I going to use this function elsewhere?_

If I am, then it's a no-brainer and I would define it as a separate function right away. If not well, does the benefit overweight the cost of increasing file size and polluting the private namespace?

Reply:
Good questions. In fact, sometimes it's opposite - extracting a function it can  even reduce the compiled code, because we don't need to carry the context. So depending on the compiler. 

Again - function is private, and it's used. Going further with your argument we would prefer to have big functions, which, IMHO is not good.

So, I don't see any benefit. I would see some if the _parent_ function would be smaller and less nested.